tests/experimental/evals/functions/test_classify.py

```diff
@@ -1193,6 +1193,62 @@ def test_run_evals_produces_expected_output_when_llm_outputs_unexpected_data(
     )
 
 
+@pytest.mark.respx(base_url="https://api.openai.com/v1/chat/completions")
+def test_run_evals_fails_gracefully_on_error(
+    respx_mock: respx.mock,
+    relevance_evaluator: LLMEvaluator,
+) -> None:
+    for matcher, response in [
+        (
+            M(content__contains="Paris is the capital of France."),
+            "relevant-explanation\nrelevant",  # missing delimiter
+        ),
+        (
+            M(content__contains="Munich is the capital of Germany."),
+            "some-explanation\nLABEL: unparseable-label",  # unparseable-label
+        ),
+        (
+            M(content__contains="Washington, D.C. is the capital of the USA."),
+            "unrelated-explanation\nLABEL: unrelated",  # normal
+        ),
```

Comment:
Shouldn't this result in a non-None output?

Reply:
Nevermind, missed the side effect.

Reply:
Might help to remove the comments, which were a little misleading in this test case.